index.js

```diff
@@ -631,6 +631,7 @@ export function listComponents(
   const compMap = {};
   const isRootPkg = ptype === "npm";
   if (Array.isArray(pkg)) {
+    pkg.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Maybe sort by purl which also includes group? Also there could be a comparator function used somewhere